xbmc/addons/kodi-addon-dev-kit/include/kodi/libXBMC_pvr.h

```diff
@@ -27,29 +27,78 @@
 #include "xbmc_pvr_types.h"
 #include "libXBMC_addon.h"
 
-#define PVR_HELPER_DLL_NAME XBMC_DLL_NAME("pvr")
-#define PVR_HELPER_DLL XBMC_DLL("pvr")
-
 #define DVD_TIME_BASE 1000000
 
 //! @todo original definition is in DVDClock.h
 #define DVD_NOPTS_VALUE 0xFFF0000000000000
 
+namespace V1
+{
+namespace KodiAPI
+{
```

Comment:
I think KodiAPI should be the outer namespace as it won't change and have the version as the second level